acceptance/analyzer_test.go

```diff
@@ -271,7 +271,7 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 				output, err := cmd.CombinedOutput()
 
 				h.AssertNotNil(t, err)
-				expected := "ensure registry read access to some-run-image-from-run-toml"
```

Comment:
The error the user might face changed, but  that shouldn't be an issue, right?

Reply:
That is fine! The new error messages shows the feature is working as intended :)